tensorflow-framework/src/main/java/org/tensorflow/framework/activations/ReLU.java

```diff
@@ -0,0 +1,143 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.activations;
+
+import org.tensorflow.DataType;
+import org.tensorflow.Operand;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.math.Greater;
+import org.tensorflow.op.nn.LeakyRelu;
+import org.tensorflow.types.family.TNumber;
+
+/**
+ * Rectified Linear Unit(ReLU) activation.
+ *
+ * <p>With default values, this returns the standard ReLU activation: <code>max(x, 0)</code>, the
+ * element-wise maximum of 0 and the input tensor.
+ *
+ * <p>Modifying default parameters allows you to use non-zero thresholds, change the max value of
+ * the activation, and to use a non-zero multiple of the input for values below the threshold.
+ *
```

Comment:
Could we adapt this part of the Keras documentation?

**alpha**: A float that governs the slope for values lower than the threshold.

**max_value**: A float that sets the saturation threshold (the largest value the function will return).

**threshold**: A float giving the threshold value of the activation function below which values will be damped or set to zero.

Reply:
I am not sure there is an advantage in Java to specify a _"A float"_ in the JavaDoc comment, as it is already defined as a `float` in the method signature. I could see in Python why this may be necessary.

Reply:
For sure! But the text like "the threshold value of the activation function below which values will be damped or set to zero" may be helpful.